lib/service/Service.ts

```diff
@@ -33,18 +37,20 @@ class Service {
    * Get general information about walli-server and the nodes it is connected to
    */
   public getInfo = async (): Promise<WalliInfo> => {
-    const { btcdClient, lndClient, ltcdClient } = this.serviceComponents;
+    const { btcdClient, lndClient, xudClient, ltcdClient } = this.serviceComponents;
```

Comment:
I would prefer the same order as in `ServiceComponents`: `{ btcdClient, ltcdClient, lndClient, xudClient }`